src/index.js

```diff
@@ -1,11 +1,12 @@
 import MascotBot from './bots/mascot-bot.js';
-import GreetOnStart from './behaviors/greet-on-start.js';
+import TestBehavior from './behaviors/test-behavior.js';
```

Comment:
TestBehavior is the same thing, just renamed. I guess it didn't get added in the commit.